stylecop.ruleset

```diff
@@ -33,7 +33,7 @@
     <Rule Id="SA1024" Action="Error" /> <!-- Colons must be spaced correctly -->
     <Rule Id="SA1025" Action="Error" /> <!-- Code must not contain multiple whitespace in a row -->
     <Rule Id="SA1026" Action="Error" /> <!-- Code must not contain space after new keyword in implicitly typed array allocation -->
-    <Rule Id="SA1027" Action="Error" /> <!-- Use tabs correctly -->
+    <Rule Id="SA1027" Action="None" /> <!-- Use tabs correctly -->
```

Comment:
This might be the culprit for many of the tab/space problems.

Reply:
No, it's really not. This is something that throws up when .editorconfig is followed and spaces are changed to tabs by the IDE. Although tabs are used correctly - and in accordance with .editorconfig - it raises this error on every single line of the file.

Reply:
Well then it kind of is, if you had changed your ide settings (ideally per project) to use spaces then this rule would work fine.
Apparently the .editorconfig was changed 29 days ago to have tabs instead of spaces, that was a review miss that that got merged.

Reply:
Why would I change my ide settings to use spaces when it's reading its settings from .editorconfig? :)

Reply:
Well I missed the fact that we had an editorconfig file.